components/unit/Pagination.vue

```diff
@@ -1,46 +1,44 @@
 <template>
-  <div v-if="!hidePagination" class="unit-pagination">
+  <div class="unit-pagination">
     <BasicLink
       v-if="previousUnit"
-      :to="`/course/${activeCourseSlug}/${previousUnit.slug}`"
+      :to="`/course/${activeCourse?.slug}/${previousUnit.slug}`"
```

Comment:
```suggestion
      :to="`/course/${activeCourse!.slug}/${previousUnit.slug}`"
```

I think we should assume that the unit exits (with the `!` instead of `?`).

And if not, we should control what to render if there is no course.
But I believe that it cannot be the case that there is no course here.

Reply:
If I replace `?` with `!` I get a lint error: `[vue/no-parsing-error]
Parsing error: Unexpected token !. eslint-plugin-vue`

Reply:
Oh, I can't replicate it:

<img width="754" alt="Screenshot 2022-12-21 at 8 47 37 AM" src="https://user-images.githubusercontent.com/1007051/208920288-7839aeed-587d-4201-859f-92927011c223.png">

Reply:
Fixed!! e290196d56aa52355886819347e4046508ae99dd